index.js

```diff
@@ -19,7 +19,7 @@ var randomBytes = require('random-bytes')
  * @private
  */
 
-var EQUAL_GLOBAL_REGEXP = /=/g
+var EQUAL_GLOBAL_REGEXP = /=+$/
```

Comment:
Remove the word global from the variable name since the regexp is no longer in global mode, please.